scripts/build_android_libtorch_and_pytorch_android.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# Must be run from the root directory of pytorch repo
+set -eux
+
+PYTORCH_DIR=$(pwd)
+PYTORCH_ANDROID_DIR=$PYTORCH_DIR/android
+WORK_DIR=$(pwd)
+
+echo "PYTORCH_DIR:$PYTORCH_DIR"
+echo "WORK_DIR:$WORK_DIR"
+
+echo "ANDROID_HOME:$ANDROID_HOME"
+if [ ! -z "$ANDROID_HOME" ]; then
+  echo "ANDROID_HOME not set; please set it to Android sdk directory"
+fi
+
+if [ ! -d $ANDROID_HOME ]; then
+  echo "ANDROID_HOME not a directory; did you install it under $ANDROID_HOME?"
+  exit 1
+fi
+
+echo "ANDROID_NDK:$ANDROID_NDK"
+if [ ! -z "$ANDROID_NDK" ]; then
+  echo "ANDROID_NDK not set; please set it to Android sdk directory"
+fi
+
+if [ ! -d $ANDROID_NDK ]; then
+  echo "ANDROID_NDK not a directory; did you install it under $ANDROID_NDK?"
+  exit 1
+fi
+
+GRADLE_PATH=gradle
+GRADLE_NOT_FOUND_MSG="Unable to find gradle, please add it to PATH or set GRADLE_HOME"
+
+if [ ! -x "$(command -v gradle)" ]; then
+  if [ -z "$GRADLE_HOME" ]; then
+    echo GRADLE_NOT_FOUND_MSG 
+    exit 1
+  fi
+  GRADLE_PATH=$GRADLE_HOME/bin/gradle
+  if [ ! -f "$GRADLE_PATH" ]; then
+    echo GRADLE_NOT_FOUND_MSG 
+    exit 1
+  fi
+fi
+echo "GRADLE_PATH:$GRADLE_PATH"
+
+ABI_ARM32=armeabi-v7a
+ABI_ARM64=arm64-v8a
+ABI_X86=x86
+ABI_X86_64=x86_64
+
+ABIS_LIST="$ABI_X86 $ABI_X86_64 $ABI_ARM32 $ABI_ARM64"
+CUSTOM_ABIS_LIST=false
+if [ $# -gt 0 ]; then
+  ABIS_LIST=$1
+  CUSTOM_ABIS_LIST=true
+fi
+  
+echo "ABIS_LIST:$ABIS_LIST"
+
+LIB_DIR=$PYTORCH_ANDROID_DIR/pytorch_android/src/main/jniLibs
+INCLUDE_DIR=$PYTORCH_ANDROID_DIR/pytorch_android/src/main/cpp/libtorch_include
+mkdir -p $LIB_DIR
+mkdir -p $INCLUDE_DIR
+
+for abi in $ABIS_LIST
+do
+echo "abi:$abi"
+
+OUT_DIR=$WORK_DIR/build_android_$abi
+
+mkdir -p $OUT_DIR
+rm -rf $OUT_DIR/*
+
+python setup.py clean
+ANDROID_ABI=$abi
+$PYTORCH_DIR/scripts/build_android.sh \
+  -DANDROID_ABI=$abi \
+  -DANDROID_CCACHE=$(which ccache) \
+  -DBUILD_CAFFE2_MOBILE=OFF \
+  -DCMAKE_PREFIX_PATH=$(python -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())') \
+  -DPYTHON_EXECUTABLE=$(python -c 'import sys; print(sys.executable)')
+
+cp -R $PYTORCH_DIR/build_android/install/lib $OUT_DIR/
+cp -R $PYTORCH_DIR/build_android/install/include $OUT_DIR/
+
+echo "$abi build output lib,include copied to $OUT_DIR" 
+
+LIB_LINK_PATH=$LIB_DIR/$abi
+INCLUDE_LINK_PATH=$INCLUDE_DIR/$abi
+
+rm -f $LIB_LINK_PATH
+rm -f $INCLUDE_LINK_PATH
+
+ln -s $OUT_DIR/lib $LIB_LINK_PATH
+ln -s $OUT_DIR/include $INCLUDE_LINK_PATH
+
+done
+
+# if need specify proxy params
+PROXY_HOST=
+PROXY_PORT=
+
+if [ -z "$PROXY_HOST" ]; then
```

Comment:
gradle proxy can be set via property file, e.g.: ~/.gradle/gradle.properties
```
systemProp.http.proxyHost=...
systemProp.http.proxyPort=8080
systemProp.https.proxyHost=...
systemProp.https.proxyPort=8080
```

Reply:
I removed proxy params from scripts, just left a bash comment with lines to specify in ~/.gradle/gradle.properties